deep-learning/_posts/2017-08-21-instance-salient-object.md

```diff
@@ -0,0 +1,59 @@
+---
+layout: review
+title:  "Instance-Level Salient Object Segmentation"
+tags:   deep-learning segmentation CNN
+author: Zhiming Luo
+pdf:    http://hcp.sysu.edu.cn/wp-content/uploads/2017/07/1704.03604.pdf
+cite:
+  authors: "G. Li, Y. Xie, L. Lin and Y. Yu"
+  title:   "Instance-Level Salient Object Segmentation"
+  venue:   "CVPR 2017"
+---
+
+Instead of only doing salient object detection, this paper also segment different salient objects into different instances.
+
+![](/deep-learning/images/salient-instance/idea.jpg)
+
+# Contributions
+
+1. A new dataset (1000 images of instance-level salient objects).
+2. A multi-scale segmenation networks. 
+
+# Framework
+![](/deep-learning/images/salient-instance/framework.jpg)
+
+1. A multi-scale segmentation network is used to compute the saliency map and contours.
+2. The multiscale combinatorial grouping (MCG) algorithm [[1]](https://www2.eecs.berkeley.edu/Research/Projects/CS/vision/grouping/mcg/resources/MCG_CVPR2014.pdf) was used to generate object proposals, and the subset optimization method [[2]](http://users.eecs.northwestern.edu/~xsh835/assets/cvpr2016_sod.pdf) selected the final salient proposals. 
+3. A fully connected CRF step [[3]](https://arxiv.org/abs/1210.5644) was adopted to get the final instance segmenation. 
+
+### 1. Multiscale segmentation network
+>- A U-Net like network architecture
+>- fusing segmenation result of multi-scale
+
+![](/deep-learning/images/salient-instance/msnet_1.jpg)
+
+### 2. Proposal step
+
+>- 800 salient object proposals for any given image based on the contour map.
```

Comment:
Please remove all the `>` in section 2 and 3. Replace `>>-` by 4 spaces followed by `-`.